@navikt/core/react/src/progress-bar/ProgressBar.tsx

```diff
@@ -0,0 +1,126 @@
+import cl from "clsx";
+import React, { HTMLAttributes, forwardRef, useEffect, useState } from "react";
+
+interface ProgressBarPropsBase
+  extends Omit<HTMLAttributes<HTMLDivElement>, "role"> {
+  /**
+   * Changes height.
+   * @default "medium"
+   */
+  size?: "large" | "medium" | "small";
+  /**
+   * Current progress.
+   */
+  value: number;
```

Comment:
Hvis value ikke har noen effekt når du har brukt duration, burde det kanskje stå noe om det i JSDOCen.

Reply:
Godt poeng. Endret også litt på ordlyden til duration for å indikere at `duration` fører til en preset animasjon.